packages/cli/src/commands/link/linkAssets.js

```diff
@@ -32,4 +38,33 @@ const linkAssets = (
   logger.info('Assets have been successfully linked to your project');
 };
 
-export default linkAssets;
+function linkAssetsCommand(
+  _: Array<string>,
+  ctx: ContextT,
+  opts: LinkFlagsType,
+) {
+  try {
+    const {platforms, project} = getPlatformsAndProject(ctx, opts);
+    const projectAssets = getAssets(ctx.root);
+
+    return linkAssets(platforms, project, projectAssets);
+  } catch (err) {
+    throw new ReactNativeNotFound();
```

Comment:
this is quite brutal and eats all errors. The `ReactNativeNotFound` check should live inside `linkAssets`, same as in `link`.

Reply:
True. Didn't notice that the `Promise.reject` call had `err` carried over.